extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -120,6 +120,31 @@ export class AmpAnalytics extends AMP.BaseElement {
           'config. No analytics data will be sent.');
       return Promise.resolve();
     }
+    if (this.config_['customVars'] && this.config_['customVarsMap']) {
+      this.config_['_customVars'] = this.config_['customVars'];
```

Comment:
I'll remove this line entirely -- it was just to save a copy in case we might need it later